src/evaluation/retrieval.py

```diff
@@ -2,8 +2,7 @@
 import numpy as np
 import torch
 
-
-def retrieval_evaluation(model_video, model_text, data, multicaption=False):
+def retrieval_evaluation(model_video, model_text, data, multicaption=False, segment=False, process_segments=None):
```

Comment:
remove process_segments and segment args, instead just check for the "segments" key in the batch["meta"]